src/main/org/deidentifier/arx/metric/Metric.java

```diff
@@ -1492,6 +1492,16 @@ public String toString() {
     }
 
     /**
+     * Calculates the score.
```

Comment:
Sorry for my earlier comment. Please revert all changes related to the class MetricConfiguration and instead create a method isScoreFunctionSupported() in the abstract base class Metric. Return "false" by default and overwrite this method for all Metrics that actually do support a score function (this overwriting will probably be part of a later pull-request)